transformer/internal/amphtml/amphtml.go

```diff
@@ -19,8 +19,8 @@ import (
 
```

Comment:
Please don't include `transformer/` changes in this PR. Those changes have to originate in Google and get synced out.

(Also, don't worry about header order errors in `transformer/` -- those are a result of the directory name transformations that happen as part of the sync.)